packages/quantic/force-app/main/default/lwc/quanticNumericFacet/__tests__/quanticNumericFacet.test.js

```diff
@@ -79,19 +143,11 @@ function prepareHeadlessState() {
       buildNumericFilter: functionsMocks.buildNumericFilter,
       buildSearchStatus: functionsMocks.buildSearchStatus,
       buildFacetConditionsManager: functionsMocks.buildFacetConditionsManager,
+      loadNumericFacetSetActions: functionsMocks.loadNumericFacetSetActions,
     };
   };
 }
 
-// Helper function to wait until the microtask queue is empty.
-function flushPromises() {
-  // eslint-disable-next-line @lwc/lwc/no-async-operation
-  return new Promise((resolve) => setTimeout(resolve, 0));
-}
-
-const exampleEngine = {
-  id: 'dummy engine',
-};
 let isInitialized = false;
 
 function mockBueno() {
```

Comment:
Should this also be added to the test utils? 

Do we often need to modify the result of `getBueno` in an eventual generic mock?

Probably out of scope for this PR, but just wondering.

Reply:
Good point yes, it would make sense to add it!